user_management_service/app/controllers/implicits/Json.scala

```diff
@@ -0,0 +1,13 @@
+package controllers.implicits
+
+import org.joda.time.DateTime
```

Comment:
Why do you need Joda time? Since JDK 8 it's available within Java's SDK

Reply:
org.joda.time.DateTime is used by Silhouette internals for session token expiration time.
The Token case class exists just to serialize token-related data provided by Silhouette to JSON and immediately send it to the user. I think that reformatting DateTime to java alternative before doing that would be unnecessary and expensive.

Reply:
The alternative is to use String here, but I think it makes it less obvious what our intentions are with this variable and what is the format of the datetime data in it.

Reply:
Take a decision on your own then.